yt/fields/fluid_fields.py

```diff
@@ -56,6 +57,31 @@ def _cell_mass(field, data):
     )
     registry.alias((ftype, "mass"), (ftype, "cell_mass"))
 
+    # momentum
+    def momentum_xyz(v):
+        def _mom(field, data):
```

Comment:
Would this be better as `_momm` to match `_momd`?

Reply:
Yes!